hcl2template/types.packer_config.go

```diff
@@ -100,11 +100,10 @@ func (c *PackerConfig) parseLocalVariables(f *hcl.File) ([]*Local, hcl.Diagnosti
 					Expr: attr.Expr,
 				})
 			}
-			allLocals = append(allLocals, locals...)
```

Comment:
I simplified this one because when append will need to extend the size of a `allLocals` it will double its size ( this is the default setting ) which fill make the `make([]*Local, 0, len(attrs))` call unnecessary. We could have guessed the size of `allLocals` but I think this would be premature optimisation and I would rather have easy to read code :D

Reply:
makes sense! thankss